### PR TITLE
Quality update 3

### PR DESCRIPTION
**Pull request by x9136**
I round corners, simplify shapes, correct sizes, and change the fill to stroke.

### CODE REVIEW

**Comment by KTibow** [comment]
fyi while `among_us` doesn't need any redrawing it could easily be converted to a stroke

**Comment by x9136** [comment]
> fyi while `among_us` doesn't need any redrawing it could easily be converted to a stroke

This icon is a low priority. If I wanted to make all icons fit the guides, I'd have to redo 3 times as many icons as I plan to.

**Comment by KTibow** [comment]
okay, in that case I might redraw some icons myself

File: svgs/fix_price.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="192" height="192" fill="none"><path stroke="#000" stroke-linejoin="round" stroke-width="12" d="M80.662 170H41.195V22h44.4l.007 35.12h-4.86l-.08 23.094h4.934v35.52h-4.934zm70.241-89.79h-50.788v35.528h50.79zm1.903-58.177h-52.65v35.055h52.65z"/></svg>
\ No newline at end of file
+<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg"><path opacity=".9" d="M107 22v34m0 30v34m44-70V28a6 6 0 0 0-6-6H49a6 6 0 0 0-6 6v136a6 6 0 0 0 6 6h28a6 6 0 0 0 6-6v-38a6 6 0 0 1 6-6h48a6 6 0 0 0 6-6V92a6 6 0 0 0-6-6H89a6 6 0 0 1-6-6V62a6 6 0 0 1 6-6h56a6 6 0 0 0 6-6Z" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/></svg>
```
Comment (Pukimaa) [inline svgs/fix_price.svg:1]: Same reason here. Looks too generic without the key aspects of the original icon.

File: svgs/intesa_sanpaolo.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="192" height="192" fill="none" stroke="#000"><path stroke-width="12" d="M30 40a10 10 0 0 1 10-10h112c5.52 0 10 4.48 10 10v112c0 5.52-4.48 10-10 10H40c-5.52 0-10-4.48-10-10z"/><path stroke-linecap="round" stroke-linejoin="round" stroke-width="8" d="M70.68 123.37V97.23c-.54-7.11-6.08-12.6-12.74-12.6s-12.2 5.48-12.74 12.6v26.13m63.47.01V97.23c-.54-7.12-6.08-12.6-12.74-12.6s-12.2 5.48-12.74 12.6v26.13m63.47.01V97.23c-.54-7.12-6.08-12.6-12.74-12.6s-12.2 5.48-12.74 12.6v26.13M146.71 72H45.29"/></svg>
\ No newline at end of file
+<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M22 58h148M22 98v36m58-36v36m58-36v36M54 98v36m58-36v36m58-36v36M54 98a16 16 0 1 0-32 0m90 0a16.002 16.002 0 0 0-16-16 16 16 0 0 0-16 16m90 0a16.002 16.002 0 0 0-16-16 16.002 16.002 0 0 0-16 16" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/></svg>
```
Comment (Pukimaa) [inline svgs/intesa_sanpaolo.svg:1]: The original icon features the box around the main part. So I think it should be kept.

**Review by Pukimaa — CHANGES_REQUESTED** [review]

**Review by Chefski — APPROVED** [review]

**Pull request merged**